javascript-practice/src/js/constants/message.js

```diff
@@ -16,6 +16,7 @@ const NOTIFY_MESSAGE = {
   UPDATE_USER_FAILED: 'Failed to update user!',
   DELETE_USER_SUCCESS: 'Delete user successfully!',
   DELETE_USER_FAILED: 'Failed to delete user!',
+  EMAIL_EXISTS: 'Email already exists or is incorrect! Please try again',
```

Comment:
Message is incorrect. You do not handle for incorrect email in this case. Please check it carefully. User can be misunderstanding

Reply:
i done fix